[2025-08-08 02:50:33,824] ERROR in logger (/Users/arlonwilber/Library/CloudStorage/GoogleDrive-awilber@wiredtriangle.com/Shared drives/AW/Personal/Projects/Bolaquent/utils/logger.py:117): Application error: {'error': '404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.', 'error_type': 'NotFound', 'context': '404_error', 'url': 'http://localhost:5010/favicon.ico', 'method': 'GET', 'ip': '127.0.0.1', 'session': {}, 'timestamp': '2025-08-08T06:50:33.824146'}
